src/pages/App.js

```diff
@@ -78,22 +101,7 @@ export default function App() {
           if (!set.has(elem.description.toLowerCase())) {
             set.add(elem.description.toLowerCase());
             return (
-              <>
-                <li className="foodDescription">{elem.description}</li>
-                <p className="title"> These are the nutrients that are present: </p>
-                <div className="listofNutration">
-                  {elem.foodNutrients.map((element) => (
-                    <p className="nutration">
-                      {element.nutrientName}
-                      =
-                      &nbsp;
-                      {element.value}
-                      {element.unitName}
-                    </p>
-                  ))}
-
-                </div>
-              </>
+              <FoodCard items={elem} />
```

Comment:
Good job making this simply into a component. Everything looks good so far and I have no complaints I will merge soon.